root-ruby-style.gemspec

```diff
@@ -1,14 +1,20 @@
 lib = File.expand_path("lib", __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
+# rubocop:disable RootCops/UseEnvvars
+
 Gem::Specification.new do |gem|
   gem.name          = "root-ruby-style"
-  gem.version       = "0.0.2"
+  gem.version       = "0.0.3"
   gem.authors       = ["Root Devs"]
   gem.email         = ["devs@joinroot.com"]
 
   gem.summary       = "Root's Ruby/Rails Style Guide"
 
+  if ENV["BUILDKITE"]
```

Comment:
Maybe this is a bit odd, but I was trying to stick to the URL setup by the build environment and it only ever gets published through CI